obsgit/obsgit.py

```diff
@@ -1551,3 +1551,7 @@ async def import_(args, config):
         loop = asyncio.get_event_loop()
         loop.run_until_complete(args.func(args, config))
         loop.close()
+
+
+if __name__ == "__main__":
```

Comment:
IMHO this is not necessary? If the entry point is pointing main(), do we want this as an executable?

Reply:
Only if in devel you want to run `python obsgit/obsgit.py`, I find it useful, but it could be removed as well.